Kyma docs - fixes and improvements (#2709)
kyma-project/kyma: docs/kyma/docs/08-02-sample-service-deployment-to-cluster.md

## Steps

### Get the kubeconfig file and configure the CLI

Follow these steps to get the `kubeconfig` file and configure the CLI to connect to the cluster:

1. Access the Console UI of your Kyma cluster.
2. Click **Administration**.
3. Click the **Download config** button to download the `kubeconfig` file to a selected location on your machine.
4. Open a terminal window.

> 5. Export the `KUBECONFIG` environment variable to point to the downloaded `kubeconfig`. Run this command:

Reviewer: ```suggestion
5. Export the **KUBECONFIG** environment variable to point to the downloaded `kubeconfig`. Run this command:
```

  ```
  export KUBECONFIG={KUBECONFIG_FILE_PATH}
  ```
  >**NOTE:** Drag and drop the `kubeconfig` file in the terminal to easily add the path of the file to the `export KUBECONFIG` command you run.

6. Run `kubectl cluster-info` to check if the CLI is connected to the correct cluster.

### Set the cluster domain as an environment variable

> The commands in this guide use URLs that require you to provide the domain of the cluster which you are using.

Reviewer: ```suggestion
The commands in this guide use URLs in which you must provide the domain of the cluster that you use.
```
